### PR TITLE
Badge on text

### PR DESCRIPTION
**Pull request by GuptaEesh**
Badge-on-text contains:
>Types
1. small badge on text
2. medium badge on text
3. large badge on text

badgeOnText

### CODE REVIEW

File: components/badge-on-text.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+.badge-on-text {
+    background-color: var(--secondary-300);
+    border-radius: 10px;
+    color: var(--white);
+    font-weight: bold;
+    min-height: 1rem;
+    min-width: 1em;
+    padding: 2px;
```
Comment (shivampandey0) [inline components/badge-on-text.css:16]: variables can be used for standard units to keep the consistency.

File: components/badge-on-text.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    top: -0.8rem;
+} */
+
+.large-badge {
+    min-height: 1.8rem;
+    min-width: 1.6em;
+    padding: 8px;
+    right: -8px;
+    top: -1.2rem;
+}
```
Comment (shivampandey0) [inline components/badge-on-text.css:37]: newline at the end of the file is recommended.
https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b

**Review by shivampandey0 — COMMENTED** [review]
Added some suggestions

**Review by ZuberDunge — COMMENTED** [review]
looks good bro

File: components/badge-on-text.css (hunk 1 above)
Comment (GuptaEesh) [reply components/badge-on-text.css:16]: Will look into that once I am done with submitting.
>And thank you for your valuable suggestion.

File: components/badge-on-text.css (hunk 2 above)
Comment (GuptaEesh) [reply components/badge-on-text.css:37]: Sure

**Pull request merged**